2048/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM nginx:latest
+
+RUN apt-get update
+RUN apt-get install -y zip curl
+
+RUN echo "daemon off;" >> /etc/nginx/nginx.conf
```

Comment:
Is this necessary? See https://github.com/nginxinc/docker-nginx/blob/11fc019b2be3ad51ba5d097b1857a099c4056213/mainline/jessie/Dockerfile#L27 for an alternative.

The whole installation process can be done in a single RUN instruction too - I don't think WORKDIR makes sense at this point.